src/components/course/enrollment/utils.js

```diff
@@ -31,6 +32,12 @@ export function determineEnrollmentType({
 
   if (userHasSubsidyRequestForCourse) { return HIDE_BUTTON; }
 
+  if (features.FEATURE_BROWSE_AND_REQUEST
+      && !hasLicenseSubsidy(userSubsidyApplicableToCourse)
```

Comment:
Should this also include whether the enterprise customer's `subsidyRequestConfiguration` is enabled, i.e. `subsidyRequestConfiguration?.subsidyRequestsEnabled`?

Reply:
Oh, yeah, good call.